File: java2csharp/java/Samsung/Look.EdgeScreen/list/CocktailListAdapterService.java
package com.example.edgescreen.list;

import android.content.Intent;
import android.widget.RemoteViewsService;

public class CocktailListAdapterService extends RemoteViewsService {

    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new CocktailListAdapterFactory(this.getApplicationContext());
    }
}

File: java2csharp/csharp/Samsung/Look.EdgeScreen/list/CocktailListAdapterService.cs
namespace com.example.edgescreen.list
{

	using Intent = android.content.Intent;
	using RemoteViewsService = android.widget.RemoteViewsService;

	public class CocktailListAdapterService : RemoteViewsService
	{

		public override RemoteViewsFactory onGetViewFactory(Intent intent)
		{
			return new CocktailListAdapterFactory(this.ApplicationContext);
		}
	}

}